src/main/resources/locales/de/plugins.json

```diff
@@ -8,6 +8,11 @@
     "info": {
       "helpText": "Sie haben Fragen zur Template Datei? Hier geht's zur",
       "documentation": "Dokumentation"
+    },
+    "templateRepository": {
+      "title": "Repository als Vorlage verwenden",
+      "start": "Template Repository",
+      "stop": "Untemplate Repository"
```

Comment:
I'm not sure with this translation. Maybe more like "Repository Vorlage erstellen"

Reply:
Or "Repository als Vorlage markieren". Might be a little too long though.